plugins/norman/resource-instance.js

```diff
@@ -150,10 +151,12 @@ export default {
   _stateRelevant() {
     if ( this.computed && this.computed.state && this.computed.state.name ) {
       return this.computed.state.name;
+    } else {
+      return this.enabled ? 'active' : 'inactive';
     }
 
     // @TODO unknown
-    return 'active';
+    // return 'active';
```

Comment:
Since you now have an else branch it should be fine to delete these comments since this section of code can't be executed.